main-app/app/scripts/core/controllers/core-controller.js

```diff
@@ -2,9 +2,10 @@
     'use strict';
     angular.module('Tombola.Games.Bingo90.Core')
 
-        .controller('MainController', ['$scope','LoginApi','GameApi','LoginProxy',  function($scope, loginApi, gameApi, loginProxy) {
+        .controller('MainController', ['$scope','LoginApi','GameApi','LoginProxy', 'GameProxy',  function($scope, loginApi, gameApi, loginProxy, gameProxy) {
             $scope.gameTitle = 'tombola';
```

Comment:
Is this likely to change? 
Not sure why you'd want to model this - better as a string in the HTML, or if you want the title to change in a lightweight provider which does this for you.